tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,84 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`conditional-expression.js 1`] = `
+<div>
+  {isCool ? "cool" : "very nice"}
+</div>;
+
+// should get formatted JSX style
+const a = something ? "string" : <div />;
+
+// should not get formatted JSX style
+const a = something ? "string" : "other string";
+
+<div>
+  {reallyLongChainOfConditionalExpressions ? (
+    "why"
+  ) : yes ? (
+    "it"
+  ) : is ? (
+    "!"
+  ) : (
+    ":)"
+  )}
+</div>;
+
+<div>
+  {reallyLongChainOfConditionalExpressions ? (
+    <span>why</span>
+  ) : yes ? (
+    <span>it</span>
+  ) : is ? (
+    <span>!</span>
+  ) : (
+    <span>:)</span>
+  )}
+</div>;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div>
+  {isCool ? (
```

Comment:
I made the decision to *always* break conditional expressions that use the JSX-style formatting. The motivation behind this was to make sure that the editor workflow described by @ryanflorence in #2208 would always be available.

I was mostly trying to avoid a situation like this:
```js
isCool ? <span>too small to break</span> : (
  <div>something that is long enough to break</div>
)
```
Which would be similar to allowing the following situation:
```js
if (isCool) "too small to break" else {
  "something that is long enough to break"
}
```
However, thinking about it again now, I could achieve the desired behavior using `group`/`ifBreak` just fine. If I made that change, this conditional expression would stay on one line (without breaking), but heterogeneous breaking (between the consequent and alternate) would be disallowed.